package.json

```diff
@@ -1130,10 +1130,14 @@
     "regenerator-runtime": "^0.13.9"
   },
   "dependencies": {
+    "@testing-library/react": "^16.0.1",
+    "@testing-library/react-hooks": "^8.0.1",
```

Comment:
This module uses `@folio/jest-config-stripes`, please use it instead of `@testing-library`.